web/src/scripts/components/menu/FilterableList.jsx

```diff
@@ -138,6 +138,11 @@ export default class FilterableList extends Component {
 
     return _.orderBy(filteredList, [
       (pkg) => {
+        if (pkg.name) {
+          return pkg.name.split('/').length
```

Comment:
Hrm, how would you do it with regex? I assume this is O(n), and doesn't have any visible performance hit.